flutter_app/lib/mavlink_communication.dart

```diff
@@ -168,4 +171,21 @@ class MavlinkCommunication {
     var frame = setMode(sequence, systemID, componentID, baseMode);
     write(frame);
   }
+
+  // Adds a waypoint
+  void sendWayPoint(int sequence, int systemID, int componentID,
+      double latitude, double longitude, double altitude) {
+    var waypointFrame = createWaypoint(
+        sequence, systemID, componentID, latitude, longitude, altitude);
+    write(waypointFrame);
+  }
+
```

Comment:
Create a function that takes the first waypoint in the queue and sends it to the done